app.py

```
@@ -142,40 +225,45 @@ def main():
     if uploaded_file:
         try:
             df = pd.read_csv(uploaded_file)
-
-            # Get unique ID numbers
             id_numbers = df['ID NUMBER'].unique()
-
-            # Prepare to collect all PDFs
             pdf_files = []
+            text_files = []
+            all_student_data = {}
```

Ruff (0.8.2)

Local variable `all_student_data` is assigned to but never used

Remove assignment to unused variable `all_student_data`

(F841)